docs/ref/contrib/postgres/operations.txt

```diff
@@ -50,60 +50,69 @@ them. In that case, connect to your Django database and run the query
 
         This is a required argument. The name of the extension to be installed.
 
+    .. attribute:: hints
+
+        .. versionadded:: 5.3
+
+        The optional ``hints`` argument will be passed as ``**hints`` to the
+        :meth:`allow_migrate` method of database routers to assist them in
+        making routing decisions. See :ref:`topics-db-multi-db-hints` for more
+        details on database hints.
+
 ``BloomExtension``
 ==================
 
-.. class:: BloomExtension()
+.. class:: BloomExtension(**options)
```

Comment:
```suggestion
.. class:: BloomExtension(hints=None)
```

with `versionadded 6.0` as well.

Reply:
Should the init definition reflect that ? ie 
```python
class BloomExtension(CreateExtension):
    def __init__(self, hints=None):
        super().__init__("bloom", hints=hints)
```

Reply:
seems a lot of duplicated prose but I followed what was requested